R/eonet.R

```diff
@@ -129,53 +148,57 @@ earth_event <- function(status = "all",
                      "search_description" = e$description)
 
   # unite and final parse ----
-      obj <- list("Events" = events,
-                  "Sources" = sources,
-                  "Categories" = categories,
-                  "Geography" = geo,
-                  "Meta" = meta)
-  return(obj)
- }
-#' Calls EONET category webservice
+  list("Events" = events,
+       "Sources" = sources,
+       "Categories" = categories,
+       "Geography" = geo,
+       "Meta" = meta) # this will be returned
+}
```

Comment:
Explicit `return()` call not needed because in R the last value is considered as a return value.
Consider this code: 
```r
> f <- function(x) x * 2
> f(3)
[1] 6
```
There is no `return` statement inside the function.